Add M3: Interactive List functionalities

lmunozm1702 opened the pull request:
### This deliverable, includes:

- Badge with active task total counter.
- Multiple task remove (all marked as completed in checkbox), by "Clear al completed link".
- Event listener in checkbox to mark as completed/active a task.
- Store and update task in localStorage.

Update README.md with total scope of the project.

lmunozm1702 committed:
Add M3: Interactive List functionalities

Add:

- Badge with active task total counter.
- Multiple task remove (all marked as completed in checkbox), by "Clear al completed link".
- Event listener in checkbox to mark as completed/active a task.
- Store and update task in localStorage.

Update README.md with total scope of the project.

lmunozm1702 committed:
Fix stylelint errors.

codecaiine commented on src/style.css:
```diff
@@ -195,3 +203,25 @@ h1 {
 .left-div-li {
   width: 100%;
 }
+
+.red-badge {
+  font-size: smaller;
+}
+
+.badge2 {
+  position: absolute;
+  top: 3px;
+  right: 3px;
+  display: inline-block;
+  min-width: 10px;
+  padding: 2px 3px;
+  font-size: 9px;
+  font-weight: 400;
+  line-height: 1;
+  color: #fff;
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: baseline;
+  background-color: red;
+  border-radius: 10px;
+}
```
- [x] Good job so far your project is impressive. It works as expected :100: But I would like to recommend you reduce the `width` of your todo list website displays a list that looks and behaves like the part of the minimalist project captured in the video below :arrow_down: 

#### Capture design
![68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067 (1)](https://user-images.githubusercontent.com/6615394/199602116-1bdf17f9-d775-4812-af03-24d2fc870893.jpeg)

#### Your design
![Screenshot from 2022-11-02 20-55-41](https://user-images.githubusercontent.com/6615394/199602142-1ee7cded-d717-4cc9-8f0b-a0cc65981fbe.png)

lmunozm1702 replied:
Hi! @codecaiine i set the max-with in 490 px as the real site in https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/.
Regards!

![image](https://user-images.githubusercontent.com/53025062/199603578-8421d295-ea01-45b4-9ad4-11f71ca6dee7.png)

codecaiine reviewed (changes requested):
Hi @lmunozm1702 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- App works as expected :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lmunozm1702 committed:
Fix code review issue

- I set max-with in 490 px, like the real site in https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/

KDlamini reviewed (approved):
### Hi @lmunozm1702 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!🎊 🥳 

## Highlights 🌞 

✔️  Good PR title and description
✔️  UI follows the given design
✔️  No linter errors

# Status: Approved ✔️ 

![giphyapproved](https://user-images.githubusercontent.com/38088818/152676874-3b2edc2c-a46f-4da6-b253-be859d2a2c65.gif)
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
